spec/integration/node/integration.spec.js

```diff
@@ -73,6 +78,24 @@ describe('integration', () => {
           done()
         })
       })
+
+      it('no keep alive', (done) => {
+        const httpAgent = new HttpAgent({ keepAlive: false })
+        configs.gatewayConfigs.HTTP.configure = () => ({ agent: httpAgent })
+        keepAliveHelper.callApiTwice().then(() => {
+          keepAliveHelper.verifySockets(httpAgent.sockets)
+          done()
+        })
```

Comment:
We can add this to both tests

```suggestion
        }).catch((response) => {
          done.fail(`test failed with promise error: ${errorMessage(response)}`)
        })
```